lib/cli/lib/project_types.js

```diff
@@ -19,3 +19,25 @@ export default {
   HTML: 'HTML',
   RIOT: 'RIOT',
 };
+
+export default projectTypes;
+
+export const supportedFrameworks = [
+  'react',
+  'react-native',
+  'vue',
+  'angular',
+  'polymer',
+  'mithril',
+  'riot',
+];
```

Comment:
```
ember
marko
meteor?
```

Reply:
Hmmm I just extracted this array to another file. Ember support wasn't present yet when I made this PR, but marko en meteor def should have been. I don't see them in the removed code either tho 😱 will add them I'm a few hours